src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -611,6 +611,24 @@ describe('MdAutocomplete', () => {
       });
     }));
 
+    it('should not call the default input behaviour when an option is selected with ENTER',
```

Comment:
You could shorten this to something like `should prevent the default enter key action` so it doesn't have to be on separate lines.